### PR TITLE
Add tsdb status script

### PR DESCRIPTION
**Pull request by mmazur**
[OSD-8683](https://issues.redhat.com/browse/OSD-8683)

### CODE REVIEW

File: scripts/SREP/tsdb-status/script.sh [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+# Originally from ops-sop.git/v4/utils/tsdb-status
+
+curl -G -s -k -H "Authorization: Bearer $(oc -n openshift-monitoring sa get-token prometheus-k8s)" "https://$(oc -n openshift-monitoring get routes prometheus-k8s -o=jsonpath='{.spec.host}')/api/v1/status/tsdb"
```
[the remark this answers is not in the document]
Comment (mmazur) [reply scripts/SREP/tsdb-status/script.sh]: done

**Comment by georgettica** [comment]
@wanghaoran1988 I should have added the approve label to make this pass?

**Comment by mmazur** [comment]
You're not an approver, I'm guessing the bot would've ignored you.

**Comment by georgettica** [comment]
(it added the lgtm label, so maybe if I would have added the approve it would have worked)

**Pull request merged**